LICENSE

```diff
@@ -1427,3 +1451,183 @@ This product bundles 'bsdiff' which is available under a "2-clause BSD" license.
 	 STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
 	 IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 	 POSSIBILITY OF SUCH DAMAGE.
+
+ This product bundles 'Simple-Windows-Posix-Semaphore' which is available under an Apache v2 License.
```

Comment:
As this is ALv2 it shouldn't be needed in the LICENSE.  There does not appear to be a NOTICE which would have to get incorporated.

Reply:
I always felt this was good hygiene as a practice. Further I always thought that Category A licenses needed to adhere to the attribution portion of the aforementioned license. 4.d of http://apache.org/licenses/LICENSE-2.0 led me to believe this is a smart inclusion. I assume there is other language that makes this recursive attribution assumed?

Reply:
The guidance from ASF is here http://www.apache.org/dev/licensing-howto.html#alv2-dep.  To your point, and in light of the ASF guidance, I believe we could just list the project and its version omitting the full license text?

Reply:
@apiri I added this very early on into the branch and thought I only had the first line ( also coincidentally only saw this in your comment ). Yes, totally agree that all of that text below the project and version is completely unnecessary. thanks!